src/connectors/sort-by/connectSortBy.js

```diff
@@ -100,21 +101,25 @@ export default function connectSortBy(renderFn, unmountFn) {
 
     return {
       init({ helper, instantSearchInstance }) {
-        const currentIndex = helper.getIndex();
-        const isIndexInList = find(items, item => item.value === currentIndex);
+        const initialIndex = helper.getIndex();
```

Comment:
not sure if this is needed, but we might remove these kind of functions from the helper, instead it can be read like `helper.index` I think. cc @samouss

Reply:
Yes I agree with @Haroenv. We can use `helper.state.index`.